.github/CODEOWNERS

```diff
@@ -0,0 +1,13 @@
+# Auto-assign the DS team
+* @teamshares/design-system
```

Comment:
Note: this syntax is in use across all our CODEOWNERS file, and always shows up incorrectly as an error.